pr_check.sh

```diff
@@ -1,4 +1,27 @@
-set -exv
+#!/bin/bash
 
-pipenv install -d
-flake8
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+export APP_NAME="ros"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ros"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
```

Comment:
If it wants the `resourceTemplate` name from deploy file, it should be:
```suggestion
export COMPONENT_NAME="ros-backend"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
```